<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalFactorSetupDialog</class>
 <widget class="QDialog" name="CalFactorSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalFactorSetupDialog</string>
  </property>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>305</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>305</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbHgb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>155</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>HGB</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbWbc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>65</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>WBC</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbRbc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>RBC</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leWholeBloodWbcFactor">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>65</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0000</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leWholeBloodRbcFactor">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0000</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leWholeBloodHgbFactor">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>155</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0000</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leWholeBloodPltFactor">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>245</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0000</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMcv">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>MCV</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPlt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>245</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>PLT</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leWholeBloodMcvFactor">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0000</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbWholeBlood">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>WholeBlood</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leWholeBloodWbcDate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>65</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>2017-10-28</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leWholeBloodRbcDate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>2017-10-28</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leWholeBloodHgbDate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>155</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>2017-10-28</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leWholeBloodMcvDate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>2017-10-28</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leWholeBloodPltDate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>245</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>2017-10-28</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePeripheralPltDate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>2017-10-28</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePeripheralWbcFactor">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0000</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePeripheralMcvFactor">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>205</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePeripheralHgbDate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>2017-10-28</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPeripheral">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Peripheral</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePeripheralWbcDate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>2017-10-28</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePeripheralMcvDate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>205</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>2017-10-28</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePeripheralRbcFactor">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>115</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePeripheralRbcDate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>115</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>2017-10-28</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePeripheralPltFactor">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0000</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePeripheralHgbFactor">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0000</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePrediluentHgbFactor">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>155</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0000</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePrediluentMcvFactor">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0000</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePrediluentPltFactor">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>245</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0000</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPrediluent">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Prediluent</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePrediluentWbcFactor">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>65</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePrediluentPltDate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>245</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>2017-10-28</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePrediluentMcvDate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>200</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>2017-10-28</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePrediluentRbcFactor">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePrediluentHgbDate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>155</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>2017-10-28</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePrediluentWbcDate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>65</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>2017-10-28</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePrediluentRbcDate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>2017-10-28</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPrint">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>305</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>DigitLineEdit</class>
   <extends>LineEdit</extends>
   <header>digitlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
